python/setup.py

```diff
@@ -30,7 +30,7 @@
 else:
     from setuptools import setup
     from setuptools.extension import Extension
-    kwargs = {'install_requires': ['numpy<=1.15.2,>=1.8.2', 'requests<2.19.0,>=2.18.4', 'graphviz<0.9.0,>=0.8.1'], 'zip_safe': False}
+    kwargs = {'install_requires': ['numpy<=1.15.2,>=1.8.2', 'requests>=2.20.0', 'graphviz<0.9.0,>=0.8.1'], 'zip_safe': False}
```

Comment:
requests follow semver. add a limit to `,<3`

Reply:
Done @szha